pages/home/use-tools/call-tools-directly.mdx

```diff
@@ -21,12 +21,17 @@ You still get the benefits of Arcade managing authorization and tool execution,
 
 ### Set environment variables
 
-You can find your API key in `~/.arcade/credentials.yaml`.
+You can find your Arcade API key in one of these locations:
+
+- [Arcade Dashboard](https://api.arcade.dev/dashboard/api-keys) under API Keys
+- `~/.arcade/credentials.yaml`
 
 ```bash
 export ARCADE_API_KEY=<your-api-key>
 ```
 
+> Need an Arcade API key? Visit the [How to get an API key](/home/api-keys) page to create one.
```

Comment:
Like the call-tools-with-models page, I think it makes sense to move the API key requirement to the Prerequisites section, since it's something you need to do before you follow the steps on the page.